scripts/export-answers.js

```diff
@@ -9,6 +9,13 @@ const { parquetInfo, getSyncDocId, hasLTIMetadata, getHash, schema } = require('
 
 const BUCKET = process.argv[2];     // e.g. concordqa-report-data
 const SOURCE =  process.argv[3];    // e.g. authoring.staging.concord.org
+const DATE_REGEX  = process.argv[4] // optional, e.g. 2021-05*
```

Comment:
this example is a glob, as a regex it will match `2021-055555` it will also match anything that starts with `2021-0` so `2021-03` would also match.

Reply:
D'oh. Being a sleepy-head. You're right.